Source/Core/Common/Timer.h

```diff
@@ -32,4 +36,21 @@ class Timer
   bool m_running{false};
 };
 
+class PrecisionTimer
+{
+public:
+  PrecisionTimer();
+  ~PrecisionTimer();
+
+  PrecisionTimer(const PrecisionTimer&) = delete;
```

Comment:
Rule of 5:

```
PrecisionTimer(const PrecisionTimer&) = delete;
PrecisionTimer& operator=(const PrecisionTimer&) = delete;
PrecisionTimer(PrecisionTimer&&) = delete;
PrecisionTimer& operator=(PrecisionTimer&&) = delete;
```

Reply:
The move-ctor/operator will be appropriately implicitly deleted because of the user-defined dtor/copy-ctor/op. :3